src/core/generator/tests/json_to_config_spec.rs

```diff
@@ -3,13 +3,16 @@ use std::path::Path;
 
 use serde::{Deserialize, Serialize};
 use serde_json::Value;
+use tailcall::core::config::GraphQLOperationType;
 use tailcall::core::generator::{Generator, Input};
+use tailcall::core::http::Method;
 use url::Url;
 
 #[derive(Serialize, Deserialize)]
 struct JsonFixture {
     url: String,
-    body: Value,
+    response: Value,
+    body: Option<Value>,
 }
```

Comment:
```suggestion
struct JsonFixture {
    request: Request,
    response: Response
}
```

Use `Request` and `Response` from execution_spec.

Reply:
Or consider using `Mock` from `tests::core::model`

Reply:
done